### ISSUE TITLE
A cheat sheet should be added to help with theme development

### ISSUE THREAD

**Issue by mauriciofauth** [phpmyadmin/phpmyadmin#17487]
A cheat sheet page that contains UI elements could be added to help with theme development.

A good example of a cheat sheet page to serve as a base is the [Bootstrap's Cheatsheet page](https://getbootstrap.com/docs/5.1/examples/cheatsheet/).

At least the custom components that are not available by default in Bootstrap should be present in this page.

Note: This cheat sheet page should only be available in [development mode](https://docs.phpmyadmin.net/en/latest/config.html#cfg_environment).

**Comment by adorade** [comment]
Has there been any progress on this? When I began creating the BooDark theme, I attempted to construct a cheatsheet similar to the one for Bootstrap 5. However, I eventually halted my efforts as I found myself stuck.

**Comment by williamdes** [comment]
> Has there been any progress on this? When I began creating the BooDark theme, I attempted to construct a cheatsheet similar to the one for Bootstrap 5. However, I eventually halted my efforts as I found myself stuck.

No, not yet. But it would be very cool to have a special dev page for this
